mobile/src/main/res/layout/color_picker_dialog.xml

```diff
@@ -1,28 +1,28 @@
 <?xml version="1.0" encoding="utf-8"?>
 <LinearLayout
     xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
     android:layout_width="wrap_content"
     android:layout_height="wrap_content"
     android:gravity="center_horizontal"
     android:paddingStart="16dp"
     android:paddingEnd="16dp"
-    android:paddingBottom="16dp"
+    android:paddingTop="8dp"
     android:orientation="vertical">
 
-    <com.larswerkman.holocolorpicker.ColorPicker
+    <com.flask.colorpicker.ColorPickerView
         android:id="@+id/picker"
         android:layout_width="240dp"
-        android:layout_height="240dp" />
+        android:layout_height="240dp"
+        app:alphaSlider="true"
+        app:density="8"
```

Comment:
From your screenshot, I think the wheel could be a bit larger, and density be increased as well. I'll play with that.

What's the meaning of the slider attributes?

Reply:
> What's the meaning of the slider attributes?

I think they can be removed.

Reply:
Suggestion:
- density 8 -> 12
- size 240dp -> match_parent

Reply:
> density 8 -> 12

IMO 12 is too much. What about 10?

> match_parent

Then the slider disappears. Do you have a working patch?

Reply:
The color wheel needs height=0dp + weight=1. I can post a patch for that tomorrow. With that + density 12 it looks like this:

https://www.dropbox.com/s/6vgsst3895shh4e/Screenshot_20200217-212830_openHAB.png?dl=0

Reply:
Layout diff for that screenshot:
```diff
diff --git a/mobile/src/main/res/layout/color_picker_dialog.xml b/mobile/src/main/res/layout/color_picker_dialog.xml
index 0fb1d862..ef4c3e6c 100644
--- a/mobile/src/main/res/layout/color_picker_dialog.xml
+++ b/mobile/src/main/res/layout/color_picker_dialog.xml
@@ -2,8 +2,8 @@
 <LinearLayout
     xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
-    android:layout_width="wrap_content"
-    android:layout_height="wrap_content"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
     android:gravity="center_horizontal"
     android:paddingStart="16dp"
     android:paddingEnd="16dp"
@@ -12,15 +12,17 @@
 
     <com.flask.colorpicker.ColorPickerView
         android:id="@+id/picker"
-        android:layout_width="240dp"
-        android:layout_height="240dp"
-        app:density="8"
+        android:layout_width="match_parent"
+        android:layout_height="0dp"
+        android:layout_weight="1"
+        app:density="12"
         app:wheelType="CIRCLE" />
 
     <com.google.android.material.slider.Slider
         android:id="@+id/brightness_slider"
-        android:layout_width="240dp"
+        android:layout_width="match_parent"
         android:layout_height="wrap_content"
+        android:layout_marginTop="8dp"
         android:valueFrom="0"
         android:valueTo="100" />
 </LinearLayout>
```

Reply:
Picked

Reply:
I changed the density to 10. Let's look at the feedback when the code goes live.

Reply:
You really think we'll get feedback for that, looking at the responses in the forum thread? ;-)

If 12 is too dense for you and 10 is too loose for me, we could settle at a compromise value of 11.

Reply:
I think we get feedback if the new color picker is in the beta version.

> If 12 is too dense for you and 10 is too loose for me, we could settle at a compromise value of 11.

Isn't 10 already a compromise between 8 and 12? ;)
I really have to see and test this when it's in beta, so I'd keep it at 10 for now.

Reply:
*sigh* OK then :)